### PR TITLE
Calculate

### PR DESCRIPTION
**Pull request by MateuszKwiecien**

### CODE REVIEW

File: module1/homework/calculate/calculate.hpp [hunk 1]
```diff
@@ -2,6 +2,18 @@
...
+    if(command == "subtract")
+        return std::to_string(first-second);
+    if(command == "multiply")
+        return std::to_string(first*second);
+    if(command == "divide")
+    {
+        if(second==0)
+            return "Division by 0";
+        else
+            return std::to_string(first/second);
```
Comment (mariomoore) [inline module1/homework/calculate/calculate.hpp:16]: This works fine, but you don't need `else` here, but:
```
if(second == 0)
    return "Division by 0";
return std::to_string(first / second);
```

Comment (DamiXioR) [reply module1/homework/calculate/calculate.hpp:16]: i agree with my predecessor, but in my opinion in this way it's better to use curly braces with if statement

Comment (nauka-programowania-MA) [reply module1/homework/calculate/calculate.hpp:16]: Yes, when someone is redundant we don't write this :)
Programmers are lazy :) try to write only what you really need.

**Comment by ziobron** [comment]
{ "requirement_ids": [1119, 1429], "points": [2, 5] }

**Pull request closed without merging**